app/assets/stylesheets/application.css

```diff
@@ -269,6 +269,7 @@ td.three_events, td.three_events span { background: #FFFDA6; }
 td.four_events, td.four_events span { background: #FFD9A6; }
 td.more_events, td.more_events span { background: #FFA6A6; }
 td.blackout_event, td.blackout_event span { background: #999999; color: #DDDDDD }
+td.current_day { outline: 1px solid #444444 !important; font-weight: bold}
```

Comment:
```suggestion
td.current_day { outline: 1px solid #444444; outline-offset: -0.5px; font-weight: bold}
```

I haven't used `outline` before, but it seems that the browser does some weird aliasing that causes the outline to appear to extend past the cell depending on the zoom level. Zooming in shows that it is half-overlapping which causes this aliasing:
![image](https://github.com/ABTech/tracker/assets/4472083/2ac434c6-b0ce-4c78-87bd-b7ef38cf6f02)
![image](https://github.com/ABTech/tracker/assets/4472083/c26867be-5121-4959-98b7-09791699075b)

Adding `outline-offset: -0.5px;` seems to fix it:
![image](https://github.com/ABTech/tracker/assets/4472083/f4631553-5da9-44fd-96cc-6ac5da01ecae)
![image](https://github.com/ABTech/tracker/assets/4472083/f56d85c8-770d-4153-bef7-c769ac9559da)


Also, you don't need `!important` since it isn't overriding any other styles.

Reply:
Oh wow good catch. I used border before but it couldn't cover the top, even with !important, so I switched to outline, which seems to lie above borders.